doc/source/examples/dem/small-scale-rotating-drum-post-processing/small-scale-rotating-drum-post-processing.rst

```diff
@@ -49,52 +51,33 @@ The DEM files used in this example are obtained following the `Small scale rotat
-
-First of all, we import the module to our Python script. There are two ways to do so. In the case of this example, we use the `sys <https://docs.python.org/3/library/sys.html>`_ module to import it directly from Lethe's directory:
-
-.. code-block::
-
-  import sys
-  path_to_module = '../../../contrib/postprocessing/'
-  sys.path.append(path_to_module)
-  from lethe_pyvista_tools import *
-  import matplotlib.pyplot as plt
-
-where `sys.path <https://docs.python.org/3/library/sys.html#sys.path:~:text=in%20version%203.10.-,sys.path%C2%B6,-A%20list%20of>`_ is a list of strings that specifies the search path for modules. However, one can simply copy the `lethe_pyvista_tools.py <https://github.com/lethe-cfd/lethe/tree/master/contrib/postprocessing>`_ file to the same folder as the Python post-processing script is and import it, such as:
- 
-.. code-block::
-
-  from lethe_pyvista_tools import *
-
-One third and very convenient way to always import the module without copying it or even adding the ``sys.path.append(path_to_module)`` is permanentely adding the path to the module (``/contrib/postprocessing/``) to your `PYTHONPATH <https://docs.python.org/3/library/sys_path_init.html#:~:text=The%20PYTHONPATH%20environment%20variable%20is,all%20installed%20Python%20versions%2Fenvironments.>`_.
-
-The ``*`` means that we want to import all members of lethe_pyvista_tools.
+Please, read this `documentation <../../../tools/postprocessing/postprocessing.py>`_ before jumping to the following steps.
 
 Constructing the Object
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The following step is to create an object to receive the data. In the present case, the object is called ``particles``.
+The first step is to create an object to receive the data. In the present case, the object is called ``particles``.
 
 .. code-block::
 
-  particles = lethe_pyvista_tools(case_path = ".", prm_file_name = "small-rotating-drum-dem.prm", pvd_name = "out.pvd", prefix = "mod_")
+  particles = lethe_pyvista_tools(case_path = ".", prm_file_name = "small-rotating-drum-dem.prm", pvd_name = "out.pvd", prefix = "mod_", n_procs = None)
 
 Here, the constructor ``lethe_pyvista_tools`` receives two arguments: ``case_path``, ``prm_file_name``, and the name of the ``.pvd`` file generated by the simulation. In the above code line, ``"."`` means that the ``case_path`` is the path where we currently are, but it can be any path where the case is. The ``prm_file_name`` argument takes ``"small-rotating-drum-dem.prm"``. The constructor can take 3 other arguments: ``first``, ``last``, and ``step``, standing for the first and last time-steps to be read and the interval between the time-steps, respectively. By default, ``first = 0``, ``step = 1``, and ``last`` is the last time-step of the data.
```

Comment:
```suggestion
Here, the constructor ``lethe_pyvista_tools`` receives three arguments: ``case_path``, ``prm_file_name``, and the name of the ``.pvd`` file generated by the simulation. In the above code line, ``"."`` means that the ``case_path`` is the path where we currently are, but it can be any path where the case is. The ``prm_file_name`` argument takes ``"small-rotating-drum-dem.prm"``. The constructor can take 3 other arguments: ``first``, ``last``, and ``step``, standing for the first and last time-steps to be read and the interval between the time-steps, respectively. By default, ``first = 0``, ``step = 1``, and ``last`` is the last time-step of the data.
```